lib/reel/websocket.rb

```diff
@@ -10,10 +9,11 @@ class WebSocket
 
     attr_reader :socket
     def_delegators :@socket, :addr, :peeraddr
+    def_delegators :@driver, :ping
 
     def initialize(info, connection)
-      driver_env = DriverEnvironment.new(info, connection.socket)      
-      
+      driver_env = DriverEnvironment.new(info, connection.socket)
+
       @socket = connection.hijack_socket
       @request_info = info
 
```

Comment:
After you post a new PR, and we give @jcoglan a bit of time to possibly review your changes and comment, I will merge this and cut it as `pre2` and we can see how it does in the wild then.